apps/root/src/services/earnService.ts

```diff
@@ -1037,11 +1088,7 @@ export class EarnService extends EventsManager<EarnServiceData> {
 
         modifiedStrategy.pendingTransaction = '';
 
-        const userHasRemainingFunds = modifiedStrategy.balances.some((balance) => balance.amount.amount > 0n);
-
-        if (userHasRemainingFunds) {
-          userStrategies.push(modifiedStrategy);
-        }
+        userStrategies.push(modifiedStrategy);
```

Comment:
good, we need to handle then to not show positions that dont have funds, but this is for another ticket

Reply:
Agree! I have created it here:
https://linear.app/balmy/issue/BLY-3234/filter-out-strategies-with-no-balance-from-portoflio-table